R/ccl_dm.R

```diff
@@ -97,10 +97,12 @@ col.location_address_geo <- function(df, state_fips) {
                   tract = ifelse(is.na(tract), tract2, tract),
                   address = stringr::str_to_title(address)) %>%
     dplyr::select(-c(lat2, long2, tract2)) %>%
+    check_tx_bounds(bb = bb) %>%
```

Comment:
I added these functions twice because we do the geocoding step next and it's not 100% accurate, but we also want to drop any that are wrong before so they have the potential to get geocoded.